diff --git a/sound/soc/fsl/fsl_sai.c b/sound/soc/fsl/fsl_sai.c
index f3d3d20d35d7..449af79e2640 100644
--- a/sound/soc/fsl/fsl_sai.c
+++ b/sound/soc/fsl/fsl_sai.c
@@ -27,9 +27,7 @@
 		       FSL_SAI_CSR_FEIE)
 
 static const unsigned int fsl_sai_rates[] = {
-	8000, 11025, 12000, 16000, 22050,
-	24000, 32000, 44100, 48000, 64000,
-	88200, 96000, 176400, 192000
+	48000, 96000, 192000
 };
 
 static const struct snd_pcm_hw_constraint_list fsl_sai_rate_constraints = {
