--- a/arch/arm64/boot/dts/freescale/Makefile	2022-01-23 09:12:53.000000000 +0100
+++ b/arch/arm64/boot/dts/freescale/Makefile	2022-01-31 22:24:16.810788796 +0100
@@ -82,6 +82,7 @@
 dtb-$(CONFIG_ARCH_MXC) += imx8mq-librem5-r4.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mq-tqma8mq-mba8mx.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mq-mnt-reform2.dtb
+dtb-$(CONFIG_ARCH_MXC) += imx8mq-mnt-reform2-no-hdmi.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mq-nitrogen.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mq-phanbell.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mq-pico-pi.dtb
